bin/newman.js

```diff
@@ -60,6 +60,8 @@ program
     .option('--ssl-extra-ca-certs <path>', 'Specify additionally trusted CA certificates (PEM)')
     .option('--cookie-jar <path>', 'Specify the path to a custom cookie jar (serialized tough-cookie JSON) ')
     .option('--export-cookie-jar <path>', 'Exports the cookie jar to a file after completing the run')
+    .option('-s, --skip-executions-summary',
```

Comment:
As we had discussed in the sync-up, we might need to change the option name.

Reply:
Changed to `--no-executions-summary` as discussed with @codenirvana